### ISSUE TITLE
Add Newsletter sign-up to Clubs page

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.mozilla.org#1358]
For those who don't want to register as a Mozilla Club because they are already affiliated with another organization, we want to offer the explicit opportunity to sign up to receive a monthly newsletter about Clubs. 

Suggested copy:
"Already running a similar program under another name? Sign up to receive our monthly newsletter featuring new activities and curriculum, tips and tricks, and more."

Those who sign up this way should be added to the Mozilla Learning Networks Exact Target mailing list.
 
Note: this can replace the current CTA on the /clubs page: "If you’d like to get started on your own, check out these resources"

/cc @amirad @HPaulJohnson

**Comment by amirad** [comment]
What about: "Already running a similar program under another name? You can still join our club network! Sign up to receive our monthly newsletter featuring new activities and curriculum, tips and tricks, and more."

We will still have a way to identify them in the backend right? and know how many people are in that group and email them separately if we want?

**Comment by Saallen** [comment]
adding myself here :camel:

**Comment by hannahkane** [comment]
@amirad - I was thinking about whether there's a need to include them in the site backend, in addition to having them in Exact Target, and I'm not sure there is. We should be sending emails from Exact Target (I added a question to @adamlofting's Exact Target doc to clarify that we want to be able to segment this group from the broader MLN list). What's the need to have this data stored in the site backend as well? The reason we do that for Clubs is so that we can manage the approval process (not needed in the case of these affiliated groups) and so that we can display them on the site (also not needed in this case).

**Comment by amirad** [comment]
@hannahkhane - have been going back and forth on this as well. I realized only benefit to having them there is being able to easily switch clubs from one to another which I know we will want to do.

**Comment by hannahkane** [comment]
I think we'd move someone from one mailing list to the other within Exact Target. We already have the ability to remove someone from the Clubs list on the backend. Does that cover everything?

**Comment by amirad** [comment]
I'm thinking through the process. Some potentials:
A club is no longer active and we need to take them off our active page. I go to backend and change their status to remove/deny. But I don't want to loose them? So I take their email and go into exact target and add it to the affiliates group. 

An affiliates group is ready to sign-up as a Mozilla Club. I tell them to go to the clubs page and sign up using the club signup. 

Those scenarios make sense @hannahkane

**Comment by hannahkane** [comment]
w00t!

**Comment by hannahkane** [comment]
Blocked on 1) Exact Target newsletter set-up, and 2) needing to confirm that we have programming planned for affiliates

**Comment by hannahkane** [comment]
Sounds like this is no longer needed, as we're not focusing on affiliates just yet.
